pie/utils/objects.py

```diff
@@ -96,3 +96,84 @@ def check(reaction, user):
                     with contextlib.suppress(nextcord.Forbidden):
                         await message.remove_reaction("▶️", user)
                     await message.edit(embed=self.pages[pagenum])
+
+
+class ConfirmView(nextcord.ui.View):
+    """Class for making confirmation embeds easy.
+    The right way of getting response is first calling wait() on instance,
+    then checking instance attribute `value`.
+    Attributes:
+        value: True if confirmed, False if declined, None if timed out
+        ctx: Context of command
+        message: Confirmation message
+    Args:
+        ctx: The context for translational and sending purposes.
+        embed: Embed to send.
+        timeout: Number of seconds before timeout. `None` if no timeout
+    """
```

Comment:
That description is confusing to those who don't know what it is used for. Can you add small codeblock below the arguments to show how it is used? You can see how it can be done in `pie/acl/__init__.py:acl()`.